packages/embed/src/initializers/build-options-from-attributes.spec.ts

```diff
@@ -30,6 +30,8 @@ describe('build-options-from-attributes', () => {
         source: 'string',
         medium: 'string',
         mediumVersion: 'string',
+        open: 'string',
+        openValue: 'integer',
```

Comment:
Please add new options as attributes to the element above (`wrapper.innerHTML` at line 6). For example:

```
<div id="element"
  ...
  data-tf-open="scroll"
  data-tf-open-value="30"
></div>
```

Then test it below:
```
expect(options).toEqual({
  ...
  open: 'scroll',
  openValue: 30,
})
```

Reply:
sure, I missed that